src/components/nav_bar.jsx

```diff
@@ -0,0 +1,57 @@
+import { NavLink } from 'react-router-dom';
+import logo from '../images/planet.png';
+
+const navLinks = [
+  {
+    path: '/',
+    id: 1,
+    text: 'Rockets',
+  },
+
+  { path: '/missions', id: 2, text: 'Missions' },
+
+  { path: '/profile', id: 3, text: 'Profile' },
+];
+
+function Navbar() {
+  return (
+    <header
+      className="d-flex"
+      style={{
+        height: '15%',
+        padding: '20px 0',
+        display: 'flex',
+        justifyContent: 'space-between',
+        alignItems: 'center',
+        borderBottom: '1px solid #7A7A7A',
+        position: 'sticky',
+        top: '0',
+        zIndex: '999999',
+        backdropFilter: 'blur(20px)',
+      }}
+    >
```

Comment:
- [ ] [Optional] On certain view screens your nav links breaks and it is not so presentable. It would be nice to fix this. See the image below:

![image](https://user-images.githubusercontent.com/77106962/175547475-00aab18f-7558-4f3e-9d72-1f7d75891a27.png)